test/image-blueprints/layer1-base/group2/rhel92-microshift-yminus2.image-installer

```diff
@@ -0,0 +1 @@
+rhel-9.2-microshift-4.{{ .Env.YMINUS2_MINOR_VERSION }}
```

Comment:
Why do we need installer?

Reply:
Good question, I'm not sure what I was doing with this. It might have been left over from trying to make the FDO test case work. I'll remove it.